### DISCUSSION TITLE
Is thinc-apple-ops used when training ner component?

### DISCUSSION THREAD

**Discussion by kinghuang** [explosion/spaCy#10693] [Help: Other Questions] [answered]
I'm training a `ner` component for the first time on a MacBook Pro with an Apple M1 Pro processor. I have the `thinc-apple-ops` module installed (via `spacy[apple]`). But, the training time of the `ner` component doesn't seem to differ with or without the module.

Should I expect a speedup when training the `ner` component with `thinc-apple-ops` installed? If so, is there something I need to configure to take advantage of it? The spaCy 3.2 blog post shows an [8✕ prediction speed improvement](https://explosion.ai/blog/spacy-v3-2#apple-ops) of the `de_core_news_lg` pipeline. Should I see something similar for training?

**Comment by adrianeboyd** [comment, marked as answer]
It should be similar because there's a large proportion of gemm in both: https://github.com/explosion/thinc-apple-ops#training

The only thing you might need to double-check is that you have a new enough version of `thinc`. `thinc-apple-ops` is used by default if available starting in v8.0.11.
